test/Spec.hs

```diff
@@ -80,6 +80,10 @@ tests docker0 = testGroup "Tests"
       , testProperty "parallel bad, see issue #218"
           (expectFailure (ioProperty (prop_echoParallelOK True <$> mkEnv)))
       ]
+  , testGroup "Union Find"
+      [ testProperty "sequential" prop_unionFind_sequential
```

Comment:
`import UnionFind` is missing?

Reply:
UnionFind should also be added to the .cabal file!

Reply:
Right!! Thank you!

Reply:
Sorry for multiple commits, I noticed your second message after I pushed my first commit

Reply:
No problem :-)